src/main/kotlin/gov/nist/secauto/oscal/tools/server/core/commands/OscalQueryCommand.kt

```diff
@@ -184,8 +174,6 @@ class QueryCommand : AbstractTerminalCommand() {
             .defaultModelNamespace(OscalModelConstants.NS_URI_OSCAL)
             .build());
             LOGGER.info("Compiling Metapath expression")
-            LOGGER.info("Compiling Metapath expression")
-            LOGGER.info("Compiling Metapath expression")
             val compiledMetapath: MetapathExpression = MetapathExpression.compile(expression, staticContext)
             LOGGER.info("Metapath expression compiled successfully")
```

Comment:
Non-blocking: should we not catch the exception?

```suggestion
            try {
                val compiledMetapath: MetapathExpression = MetapathExpression.compile(expression, staticContext)
                LOGGER.info("Metapath expression compiled successfully")                
            } catch (ex: ???) {
                LOGGER.error("Metapath expression did not compile", ex)
            }
```